css/color-scheme.css

```diff
@@ -441,6 +414,7 @@ body {
     align-items: center;
     gap: 1rem;
     line-height: 25px;
```

Comment:
line height can make you go mad use cautiously

Reply:
Noted :D